packages/backend/src/api/controllers/tvl/DetailedTvlController.ts

```diff
@@ -149,6 +145,72 @@ export class DetailedTvlController {
     }
   }
 
+  async getDetailedAggregatedApiResponse(
+    slugs: string[],
+  ): Promise<TvlProjectResult> {
+    const dataTimings = await this.getDataTimings()
```

Comment:
move this function above the if with condition : `!dataTimings.latestTimestamp`, this way we will not spend computations on getting the timings is the slugs are empty